rootstock-touch-install

```diff
@@ -65,14 +70,15 @@ check_mounts(){
 prepare_ubuntu_system()
 {
 	do_shell "rm -f /data/rootfs.img"
-	do_shell "dd if=/dev/zero of=/data/rootfs.img seek=500K bs=4096 count=0 >/dev/null 2>&1"
+	do_shell "dd if=/dev/zero of=/data/rootfs.img seek=511K bs=4096 count=0 >/dev/null 2>&1"
```

Comment:
?

Reply:
Makes the image as large as possible without overflowing.